src/web/twig/Extension.php

```diff
@@ -781,6 +782,19 @@ public function expressionFunction($expression, $params = [], $config = []): Exp
         return new Expression($expression, $params, $config);
     }
 
+    /**
```

Comment:
This seems unrelated to the rest of the pull request?

Reply:
yeah sorry, I must have screwed something up when I synced my fork with the remote

Reply:
That's a PR that @brandonkelly ended up renaming to `create()` -- I'd love to know what I'm doing wrong when syncing my fork with the upstream, sorry for the hassles.

Reply:
@khalwat Best bet is to make branches for each PR, so you're never committing directly to your fork’s `master` or `develop` branch - those only get pulled from upstream.

Reply:
Makes total sense, thank you @brandonkelly